pandas/core/internals/api.py

```diff
@@ -79,3 +96,31 @@ def maybe_infer_ndim(values, placement: BlockPlacement, ndim: Optional[int]) ->
         else:
             ndim = values.ndim
     return ndim
+
+
+# TODO: deprecate following https://github.com/dask/fastparquet/pull/571
```

Comment:
can you move this to a deprecated.py module (and import here)

Reply:
i guess

Reply:
looks like fastparquet will be releasing in the not-too-distant future, at which point we can bump our min version and this can be removed entirely